pkg/upgrade/upgrader/spc_upgrade.go

```diff
@@ -98,6 +98,7 @@ func updateSPCVersion(name string) error {
 		return err
 	}
 	spcObj.VersionDetails.Desired = upgradeVersion
+	spcObj.VersionDetails.Status.State = apis.ReconcilePending
```

Comment:
Pending has been set only for SPC.. why?
for other CRs as well, this need to be done.. right?

Reply:
For others the template is used to patch the CRs. The template changes are also present in this PR.